public/components/navbar.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+      crossorigin="anonymous"
+    />
+    <style>
+      a {
+        color: black;
+        text-decoration: none;
+      }
+    </style>
+    <script src="./js/home.js"></script>
+    <title>Home Page</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-expand border border-dark">
```

Comment:
I was under the impression that jQuery's `.load()` could take a file with just a subset of html tags. I believe this `navbar.html` file could just be the nav part. Like

```html
    <nav class="navbar navbar-expand border border-dark">
      <div class="container-fluid">
        <div class="collapse navbar-collapse">
          <ul class="navbar-nav me-auto">
            <li class="nav-item nav-link">
              <a href="#"> Test1 </a>
            </li>
            <li class="nav-item nav-link"><a href="#">Test2</a></li>
            <li class="nav-item nav-link"><a href="#">Test3</a></li>
          </ul>
          <div class="position-absolute end-0 d-flex align-items-center mx-5">
            <a href="../cart.html" style="text-decoration: underline"> Cart </a>
          </div>
        </div>
      </div>
    </nav>
```

and nothing else. No need for the `<html>` stuff. Could you try that? Or did you already try that and it didn't work?